src/pages/en/guides/data-fetching.md

```diff
@@ -72,33 +104,69 @@ const weather = json.data;
 <p>Weather: {weather.getCityByName.weather.summary.description}</p>
 ```
 
-:::note
-Remember, all data in Astro components is fetched when a component is rendered.
+## Fetch from a Headless CMS
 
-Your deployed Astro site will fetch data **once, at build time**. In dev, you will see data fetches on component refreshes. If you need to re-fetch data multiple times client-side, use a [framework component](/en/core-concepts/framework-components/) or a [client-side script](/en/core-concepts/astro-components/#client-side-scripts) in an Astro component.
-:::
+Fetch remote content from your favourite CMS like StoryBlok or WordPress!
 
+With an API key, some helper functions, and `getStaticPaths()` to generate dynamic page routes, an `.astro` component can fetch and render data from your CMS to generate your page content.
 
-## `fetch()` in Framework Components
+Here are some examples of what this data fetching looks like in Astro, with links to full tutorials below.
 
-The `fetch()` function is also globally available to any [framework components](/en/core-concepts/framework-components/):
-
-```tsx title="src/components/Movies.tsx" /await fetch\\(.*?\\)/
-import type { FunctionalComponent } from 'preact';
-import { h } from 'preact';
-
-const data = await fetch('https://example.com/movies.json').then((response) =>
-  response.json()
-);
+### Example: StoryBlok API
 
-// Components that are build-time rendered also log to the CLI.
-// When rendered with a client:* directive, they also log to the browser console.
-console.log(data);
+```astro
+---
+// src/pages/[slug].astro
+// Fetch a list of your StoryBlok page links using @storyblok/js and axios
+import BaseLayout from '../layouts/BaseLayout.astro';
+import { storyblokInit, apiPlugin } from "@storyblok/js";
+
+const { storyblokApi } = storyblokInit({
+  accessToken: "MY_STORYBLOK_ACCESS_TOKEN",
+  use: [apiPlugin],
+});
+
+const { data } = await storyblokApi.get('cdn/links');
+const links = Object.values(data.links);
+---
+<BaseLayout>
+  <h1>Astro + Storyblok</h1>
+  <ul>
+    {links.map(link => (
+      <li><a href={link.slug}>{link.name}</a></li>
+    ))}
+  </ul>
+</BaseLayout>
+```
+See the full tutorial [Add a Headless CMS to Astro in 5 Minutes](https://www.storyblok.com/tp/add-a-headless-cms-to-astro-in-5-minutes) to add StoryBlock to your Astro project!
 
-const Movies: FunctionalComponent = () => {
-// Output the result to the page
-  return <div>{JSON.stringify(data)}</div>;
-};
+### Example: WordPress + GraphQL
 
-export default Movies;
+```astro
+---
+// src/pages/blog/[slug].astro
+// Render pages dynamically from your Wordpress content
```

Comment:
This is the only case where we are using `Wordpress` rather than `WordPress`
```suggestion
// Render pages dynamically from your WordPress content
```